### PR TITLE
Some fixes for the test suite

### PR DESCRIPTION
**Pull request by XapaJIaMnu**
This should fix the issues i opened. It's not complete yet, I'll add some new things to it later today.

### CODE REVIEW

**Comment by XapaJIaMnu** [comment]
Should be ready now

File: Makefile [hunk 1]
```diff
@@ -28,6 +28,9 @@ pip: requirements.txt
 models:
 	mkdir -p $@
 	cd $@ && bash ./download-models.sh # $(TARBALLS)
+	cd deen/ende.student.tiny11
+	gunzip lex.s2t.gz # wasm build doesn't support zipped input
+	cd ../..
```
Comment (jerinphilip) [inline Makefile:33]: Does this work, make launches commands in different shells right? cd && chaining or paranthesis is required, or am I missing something?

Comment (XapaJIaMnu) [reply Makefile]: I forgot that it's a makefile... fixd

**Comment by jerinphilip** [comment]
Comment uncomment seems to be nits I don't really care about, merging in. Fingers crossed, we'll know if and when Jenkins breaks and follow up.

**Pull request merged**